src/prefect/cli/block.py

```diff
@@ -72,6 +73,31 @@ def display_block_type(block_type):
     return block_type_table
 
 
+def display_block_schema_properties(block_schema_fields):
+    required = block_schema_fields.get("required", [])
+    properties = block_schema_fields.get("properties", {})
+
+    block_schema_yaml_table = Table(
+        title="Schema Properties",
+        show_header=False,
+        show_footer=False,
+        show_lines=True,
+        expand=True,
+    )
+    block_schema_yaml_table.add_column(style="cyan")
+    block_schema_yaml_table.add_column()
+
+    for property_name, property_schema in properties.items():
+        if property_name in required:
+            property_schema["required"] = True
+
+        block_schema_yaml_table.add_row(
+            property_name, yaml.dump(property_schema, default_flow_style=False)
+        )
+
+    return block_schema_yaml_table
```

Comment:
https://www.loom.com/share/b47cae0cacff4bf9907e2823ebcd60d9?sid=74f80ff0-c2dc-4598-b56b-aeb34549c1cd

Reply:
we have some jsonschema `$ref` references here - originally @desertaxle and i discussed maybe parsing these to show a `prefect block type inspect <slug>` breacrumb inside, but after some trial-and-error, im realizing that:

- not all `$ref` are a block type, eg. `AwsClientParameters`, `SMTPType`, etc.
- we'd have to loop and verify each $ref to see if they're a valid block type
- _**then**_ we could hyphen-case them

i'm inclined to keep the jsonschema ref class, and we can iterate on this

Reply:
example:
![image](https://github.com/PrefectHQ/prefect/assets/19556538/d0a3177d-c7f4-4367-8fe5-4ebdb346a20d)